content.js

```diff
@@ -287,36 +287,227 @@ if (window.contentScriptInjected !== true) {
 
     const digraphExceptions = {
         "dj": [
-            "adjektiv",
-            "adjunkt",
-            "izblijedjel",
-            "nadjaha",
-            "nadjača",
-            "nadjunači",
-            "nenadjačan",
-            "odjah",
-            "odjav",
-            "odjaš",
-            "odjeb",
-            "odjeci",
-            "odjed",
-            "odjek",
-            "odjel",
-            "odjezd",
-            "odjur",
-            "ovdje",
-            "podjamči",
-            "podjar",
-            "podjarmi",
-            "podjednak",
-            "podjel",
-            "podjezič",
-            "predjel",
-            "razdjel",
-            "udjel",
-            "vindjakn",
-            "zapodjen",
-            "zdjel",
+          "adjektiv",
+          "adjunkt",
+          "bazdje",
+          "bdje",
+          "bezdje",
+          "blijedje",
+          "bludje",
+          "bridjе",
+          "vidje",
+          "vindjakn",
+          "višenedje",
+          "vrijedje",
+          "gdje",
+          "gudjer",
+          "gudje",
+          "gdjir",
+          "daždje",
+          "dvonedje",
+          "devetonedje",
+          "desetonedje",
+          "djev",
+          "djed",
+          "djejstv",
+          "djelat",
+          "djel",
+          "djelc",
+          "djelj",
+          "djenem",
+          "djeneš",
+          //"djene" rare (+ Дјене (town)), but it would colide with ђене-ђене, ђеневљанка, ђенерал итд.
+          "djenu",
+          "djet",
+          "djec",
+          "dječ",
+          "djuar",
+          "djubison",
+          "djubouz",
+          "djuer",
+          "djui",
+        // "djuk", djuk (engl. Duke) косило би се нпр. са Djukanović
+          "djuks",
+          "djulej",
+          "djumars",
+          "djupont",
+          "djurant",
+          "djusenberi",
+          "djuharst",
+          "djuherst",
+          "dovdje",
+          "dogrdje",
+          "dodjel",
+          "dodjelj",
+          "drvodje",
+          "drugdje",
+          "elektrosnabdje",
+          "žudje",
+          "zabludje",
+          "zavidje",
+          "zavrijedje",
+          "zagudje",
+          "zadjev",
+          "zadjen",
+          "zalebdje",
+          "zaludje",
+          "zaodje",
+          "zapodje",
+          "zarudje",
+          "zasjedje",
+          "zasmrdje",
+          "zastidje",
+          "zaštedje",
+          "zdjel",
+          "zdjelj",
+          "zdje",
```

Comment:
"zdje" already covers both the "zdjel" and "zdjelj" so we can remove these longer variants.